docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+| id                       | 2f347948-9460-4ac0-a588-32187c8b6ab1                                                                      |
+| ip                       | 192.168.215.98, 5.6.7.8                                                                                   |
+| name                     | pipeline_db_instance                                                                                      |
+| operating_status         | HEALTHY                                                                                                   |
+| public                   | False                                                                                                     |
+| region                   | regionOne                                                                                                 |
+| service_status_updated   | 2025-02-04T13:16:10                                                                                       |
+| status                   | ACTIVE                                                                                                    |
+| updated                  | 2025-02-04T13:16:52                                                                                       |
+| volume                   | 1                                                                                                         |
+| volume_used              | 0.08                                                                                                      |
++--------------------------+-----------------------------------------------------------------------------------------------------------+
+```
+We can see that the public IP address of our virtual machine (`1.2.3.4/32`) is indeed listed in the `allowed_cidrs`.
+
+From the information displayed with the previous command we also extract and take down the public IP address of our database instance, which we will need in the following steps.
+The first row, whose name is `addresses`, contains two parts: an address that is marked as `private`, and one that is marked as `public`.
+We are interested in the second one, so we note down `5.6.7.8`, which is a dummy value selected for the purpose of this tutorial.
+
+### Configuring access from virtual machine to Allas
+
+Similarly to our own workstation, to access the buckets hosted in Allas the virtual machine needs to be configured as well.
+
+We go back to `terminal_C`, where we are still logged in to our virtual machine in cPouta, and we follow the [instructions on how to install and configure s3cmd](../../../data/Allas/using_allas/s3_client.md#getting-started-with-s3cmd).
+It basically boils down to running the following commands:
+```
+$ sudo apt update
+
+$ sudo apt install python3-openstackclient s3cmd
+
+$ wget https://raw.githubusercontent.com/CSCfi/allas-cli-utils/master/allas_conf
+
+$ source allas_conf --mode S3 --user your-csc-username
+```
```

Comment:
This should be on the first mention of the link, not the second.

Reply:
Fix proposed in beaa887d395603c23fcc85628d8d5f9bda98bf53